### PR TITLE
add explicit test command?

### PR DESCRIPTION
**Pull request by Kidlike**
I wasn't sure if this is a good idea or not, as other tracks don't have this. But again, other tracks have more established testing concepts.

Since this is a "hello world" exercise, it could be discouraging that there are no instructions.

I hope this has not been discussed before, but I couldn't find anything except people being very confused about what they are doing wrong. :man_shrugging: 

---

Reviewer Resources:

[Track Policies](https://github.com/exercism/bash/blob/master/POLICIES.md)

### CODE REVIEW

File: exercises/practice/hello-world/.docs/instructions.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ or environment.
 The objectives are simple:
 
 - Write a function that returns the string "Hello, World!".
-- Run the test suite and make sure that it succeeds.
+- Run the test suite and make sure that it succeeds: `bats hello_world.sh`
```
Comment (glennj) [inline exercises/practice/hello-world/.docs/instructions.md:12]: Needs to be `bats hello_world.bats`

Comment (glennj) [reply exercises/practice/hello-world/.docs/instructions.md:12]: Perhaps also link to the testing documentation:
```
[Run the test suite][testing] and make sure ...

[testing]: https://exercism.org/docs/tracks/bash/tests
```

**Review by glennj — CHANGES_REQUESTED** [review]

File: exercises/practice/hello-world/.docs/instructions.md (hunk 1 above)
Comment (Kidlike) [reply exercises/practice/hello-world/.docs/instructions.md]: well, that was an embarrassing typo :)

adjusted :+1:

**Review by kotp — APPROVED** [review]
The introductory exercise does need to be a little more self contained, so I think this is good.

I like that it also links to the documentation page as well.

**Review by glennj — APPROVED** [review]

**Comment by glennj** [comment]
Thank you very much @Kidlike

**Pull request merged**